lib/open_api_spex/plug/swagger_ui.ex

```diff
@@ -33,14 +41,16 @@ defmodule OpenApiSpex.Plug.SwaggerUI do
   """
   @behaviour Plug
 
+  @swagger_ui_version Application.compile_env(:open_api_spex, :swagger_ui_version, "latest")
```

Comment:
@mbuhot @jarmo it makes sense to me to default to a recent version which is known to work instead of `latest`. Furthermore, instead of specifying the version, we can allow setting the JS and CSS URLs:

```elixir
 get "/docs/api", OpenApiSpex.Plug.SwaggerUI, 
  path: "/my_service/docs/api_spec", 
  swagger_ui_js_url: "https://cdn.jsdelivr.net/npm/swagger-ui-dist@5/swagger-ui-bundle.js",
  swagger_ui_css_url: "https://cdn.jsdelivr.net/npm/swagger-ui-dist@5/swagger-ui.css"
```

This is inspired by [fastapi](https://github.com/fastapi/fastapi/blob/master/fastapi/openapi/docs.py#L56). It also enables asset self-hosting or using the CDN of one's choice.